core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -314,7 +309,7 @@ private[spark] class BlockManager(
       val memSize = if (memoryStore.contains(blockId)) memoryStore.getSize(blockId) else 0L
       val diskSize = if (diskStore.contains(blockId)) diskStore.getSize(blockId) else 0L
       // Assume that block is not in external block store
```

Comment:
Is this comment out-of-date now that we've removed the external block store API?